files/tasks/packit-deps.yaml

```diff
@@ -5,8 +5,9 @@
 - name: Checkout packit
   git:
     clone: yes
-    repo: https://github.com/packit-service/packit.git
+    repo: https://github.com/jscotka/packit.git
     dest: "{{ reverse_dir }}"
+    version: fix_base_requre_class
```

Comment:
Will this change be removed before the merge?

Reply:
Sure, there is one commit with comment prefix  `REMOVE:` to ensure to not merge it.